pubspec.yaml

```diff
@@ -40,6 +40,9 @@ dependencies:
   image_picker: ^0.8.5+3
   sqflite: ^2.0.3+1
```

Comment:
delete unused dependencies and also place dependencies in alphabetic order, it will be more readable when it will have more dependencies